datafusion/sqllogictest/test_files/regexp.slt

```diff
@@ -354,6 +377,29 @@ X
 X
 X
 
+# test string view
```

Comment:
As discussed in https://github.com/apache/datafusion/issues/11911#issuecomment-2631559010 it would be great to move these tests into string.slt but we can totally do it as a follow on as well